packages/react-devtools-inline/__tests__/__e2e__/inspecting-props.test.js

```diff
@@ -8,9 +8,9 @@ test.describe('Testing Todo-List App', () => {
   let page, frameElementHandle, frame;
   test.beforeAll(async ({browser}) => {
     page = await browser.newPage();
-    await page.goto('http://localhost:8080/', {waitUntil: 'domcontentloaded'});
-    await page.waitForSelector('iframe#target');
-    frameElementHandle = await page.$('#target');
+    await page.goto('http://localhost:8080/e2e.html', {waitUntil: 'domcontentloaded'});
```

Comment:
Longer term, relying on our test shell for e2e tests seems too fragile. Changing one shouldn't break the other. So I added a specific e2e target.